app/controllers/admin/logs_controller.rb

```diff
@@ -0,0 +1,26 @@
+module Admin
+  class LogsController < Admin::BaseController
+    before_action :retrieve
+
+    def index; end
+
+    def show
+      @path = params[:path]
+      @result = []
+      return unless @files.include?(@path)
+
+      File.open(@path) do |file|
```

Comment:
__[Brakeman]__

Parameter value used in file name


<!-- Comment by [SideCI](https://sideci.com) -->